build_tools/benchmarks/reporting/common/utils.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import pandas as pd
```

Comment:
Pandas seems like a bit of a heavy dependency to just format a table. Python has built-in csv reader/writer (https://docs.python.org/3/library/csv.html). Maybe there are some things you're doing that would be more difficult though . Would just numpy help?

Reply:
The raw .csv input undergoes some filtering, calculations, etc. to eventually get the final HTML. Pandas has the functionality to process tabular data that the Python csv library doesn't have. Numpy doesn't have the visualization capabilities that Pandas offers. I would prefer to stick with Pandas - it makes it really easy to manipulate and visualize tabular data.

Reply:
SGTM